tests/gocase/unit/type/list/list-2_test.go

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ * Copyright (c) 2006-2020, Salvatore Sanfilippo
+ * See bundled license file licenses/LICENSE.redis for details.
+ *
+ * This file is copied and modified from the Redis project,
+ * which started out as: https://github.com/redis/redis/blob/dbcc0a8/tests/unit/type/list-2.tcl
+ */
+
+package list
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"strconv"
+	"testing"
+
+	"github.com/apache/incubator-kvrocks/tests/gocase/util"
+	"github.com/stretchr/testify/require"
+)
+
+func randInt64Str() string {
+	return strconv.FormatInt(rand.Int63(), 10)
+}
+
+func min(a, b int) int {
```

Comment:
Is it ok to place utils here, or this such things should be placed elsewhere?

Reply:
How about moving them into `https://github.com/apache/incubator-kvrocks/tree/unstable/tests/gocase/util`?

Reply:
It will be wonderful if you could implement a generic version and put into tests/gocase/util. So many pain in golang.

Reply:
I'd prefer to use an existing mature lib like https://pkg.go.dev/modernc.org/mathutil#Min.

Reply:
> I'd prefer to use an existing mature lib like https://pkg.go.dev/modernc.org/mathutil#Min.

done